src/adafruit_blinka/board/odroidn2.py

```diff
@@ -0,0 +1,154 @@
+"""Pin definitions for the Odroid N2."""
+
+from adafruit_blinka.microcontroller.amlogic.s922x import pin
+
+#GPIOX_0 = pin.GPIO228
```

Comment:
Can we remove all the commented out lines and leave the ones relevant to the N2 only?